src/drivers/drv_pwm_output.h

```diff
@@ -83,6 +83,7 @@ __BEGIN_DECLS
 #define PWM_HIGHEST_MAX 255
 #define PWM_DEFAULT_MAX 255
 #define PWM_LOWEST_MAX 255
+#define PWM_DEFAULT_TRIM 0
```

Comment:
We now have "normal" PWM, oneshot and brushed motor control lumped together into one function.
Does that introduce unnecessary complexity and greater potential problem "corner" cases? This certainly argues that the default trim value should be zero, and in "normalized" units so that no assumption needs to be made about what "PWM" means.
Also, it seems like some of these macros should be airframe-specific, not tied to a particular CONFIG_ARCH_BOARD.